operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -8,86 +8,103 @@
 # If new groups are added to these defaults, you must remember to add the merge code to vars/main.yml.
 
 kiali_defaults:
-  installation_tag: ""
-  istio_namespace: ""
+  installation_tag: ''
+  istio_namespace: ''
 
   api:
     namespaces:
       exclude:
-      - "istio-operator"
```

Comment:
why are all the indentations to the lists changing? Revert all of these changes.

Reply:
Revereted to previous